pages/goals/index.js

```diff
@@ -1,20 +1,29 @@
-import GoalList from '../components/goals/GoalList';
-import Page from '../components/Page';
+import GoalList from '../../components/goals/GoalList';
+import Page from '../../components/Page';
 import { useSession, signIn } from 'next-auth/react';
-
-//TODO to be replaced by the fetched data
-const goals = [
-  { id: 1, owner: 'Erik', title: 'Erik goal', status: 'pending' },
-  { id: 1, owner: 'Zara', title: 'Zara goal', status: 'approved' },
-];
+import GoalManagementService from '../../helpers/service/GoalManagementService';
+import { useEffect, useState } from 'react';
 
 function Goals() {
   const { data: token } = useSession();
+  const [goals, setGoals] = useState([]);
+
+  useEffect(() => {
+    const getGoals = async () => {
+      try {
+        const { data } = await GoalManagementService.getEmployeeGoals();
+        setGoals(data);
+      } catch (error) {
+        //
```

Comment:
Please handle error here, you can do console log(error) for example for simplicity